docs/src/guides/advanced/metrics.md

```diff
@@ -0,0 +1,214 @@
+---
```

Comment:
Overall I think this doc is great - very informative and readable. One thing that's maybe missing (sorry if I just missed it myself) is an explanation of how we can reference other values in a metric definition's source using `$use`? Is that still relevant/worth calling out here?

Reply:
@juliannemarik thats a whole can of worms that I'd like  you and Jordan to work out in code. 

`$use` is an option, but we need to decide when to do the resolution b/c the "path" for `$use` is dependent on the shape of the whole object when it's resolved. It's  also more complex to go from a single object, into two objects, one with a `$use` to the other. Mind-bendy... 

We could also just store the metric `id` and decide to always pass a hash of "data sources/metrics" into all cards along with their configurations. This resolves a number of other similar issues we'll need to address with the editors, such as the scenario where we want to add a Metric Card to a Site, pulling the metric value comes from an Initiative item.